src/main/kotlin/org/danilopianini/gradle/gitsemver/GitSemVerExtension.kt

```diff
@@ -113,7 +113,12 @@ open class GitSemVerExtension @JvmOverloads constructor(
                     if (!closestTag.buildMetadata.isEmpty()) {
                         logger.warn("Build metadata of closest tag $closestTag will be ignored.")
                     }
-                    val distance = runCommand("git", "rev-list", "--count", "$closestTag..HEAD")?.toLong()
+                    val distance = runCommand(
+                        "git",
+                        "rev-list",
+                        "--count",
+                        "${versionPrefix.get()}$closestTag..HEAD"
```

Comment:
```suggestion
                        "${versionPrefix.get()}$closestTag..HEAD",
```

missing trailing comma